mediator/med_phases_profile_mod.F90

```diff
@@ -170,7 +170,8 @@ subroutine med_phases_profile(gcomp, rc)
           call ESMF_TimeGet(nexttime, timestring=nexttimestr, rc=rc)
           if (med_utils_ChkErr(rc,__LINE__,u_FILE_u)) return
           ! get current wall clock time
-          call ESMF_TimeSet(wallclockTime, calkindflag=ESMF_CALKIND_GREGORIAN, rc=rc)
+          ! s=0 is to prevent an internal divide by 0 error in esmf
```

Comment:
@mvertens I think that a further optimization is possible here.   Make wallclocktime a saved variable and move lines 174-175 to the first iteration only - line 120.   I'm out today or I would try it myself.